### PR TITLE
docs(docs): improve landing docs

### PR DESCRIPTION
**Pull request by Karnaukhov-kh**
Small cleanup, minor layout changes and switch of the docs sections.

### CODE REVIEW

File: libs/state/README.md [hunk 1]
```diff
@@ -8,17 +8,86 @@
...
+```
+
+## Version Compatibility
+
+| Angular                | RxJS                 | @rx-angular/state |
+| ---------------------- | -------------------- | ----------------- |
+| `^12.0.0` or `^13.0.0` | `^6.5.5` or `^7.4.0` | `> 1.4.6`         |
+| `^11.0.0`              | `^6.5.5`             | `<= 1.4.6`        |
+
+Regarding the compatibility to RxJs, we generally stick to the compatibilities of the angular framework itself.
```
Comment (edbzn) [inline libs/state/README.md:41]:
```suggestion
Regarding the compatibility with RxJS, we generally stick to the compatibilities of the Angular framework itself.
```

File: libs/state/README.md [hunk 2]
```diff
@@ -8,17 +8,86 @@
...
+All the packages support RxJs versions `^6.5.5` || `^7.4.0`.
```
Comment (edbzn) [inline libs/state/README.md:42]:
```suggestion
All the packages support RxJS versions `^6.5.5` || `^7.4.0`.
```

File: libs/state/README.md [hunk 3]
```diff
@@ -8,17 +8,86 @@
...
+For more information about the compatibilities of angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3)
```
Comment (edbzn) [inline libs/state/README.md:43]:
```suggestion
For more information about the compatibilities of Angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3).
```

File: libs/template/README.md [hunk 4]
```diff
@@ -72,9 +45,16 @@ ng update @rx-angular/template
...
+
+| Angular                | RxJS                 | @rx-angular/template |
+| ---------------------- | -------------------- | -------------------- |
+| `^12.0.0` or `^13.0.0` | `^6.5.5` or `^7.4.0` | `> 1.0.0-beta.29`    |
+| `^11.0.0`              | `^6.5.5`             | `<= 1.0.0-beta.29`   |
 
-[API Documentation](https://github.com/rx-angular/rx-angular/tree/main/libs/template/docs/api/overview.md)
+Regarding the compatibility to RxJs, we generally stick to the compatibilities of the angular framework itself.
+All the packages support RxJs versions `^6.5.5` || `^7.4.0`.
+For more information about the compatibilities of angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3)
```
Comment (edbzn) [inline libs/template/README.md:57]:
```suggestion
For more information about the compatibilities of Angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3).
```

File: libs/template/README.md [hunk 5]
```diff
@@ -72,9 +45,16 @@ ng update @rx-angular/template
...
-## API
+## Version Compatibility
+
+| Angular                | RxJS                 | @rx-angular/template |
+| ---------------------- | -------------------- | -------------------- |
+| `^12.0.0` or `^13.0.0` | `^6.5.5` or `^7.4.0` | `> 1.0.0-beta.29`    |
+| `^11.0.0`              | `^6.5.5`             | `<= 1.0.0-beta.29`   |
 
-[API Documentation](https://github.com/rx-angular/rx-angular/tree/main/libs/template/docs/api/overview.md)
+Regarding the compatibility to RxJs, we generally stick to the compatibilities of the angular framework itself.
```
Comment (edbzn) [inline libs/template/README.md:55]:
```suggestion
Regarding the compatibility with RxJS, we generally stick to the compatibilities of the Angular framework itself.
```

File: libs/template/README.md [hunk 6]
```diff
@@ -72,9 +45,16 @@ ng update @rx-angular/template
...
+All the packages support RxJs versions `^6.5.5` || `^7.4.0`.
```
Comment (edbzn) [inline libs/template/README.md:56]:
```suggestion
All the packages support RxJS versions `^6.5.5` || `^7.4.0`.
```

File: libs/template/README.md [hunk 7]
```diff
@@ -90,31 +70,47 @@ import { LetModule, PushModule, ViewportPrioModule } from '@rx-angular/template'
...
+  - [Core concepts](https://github.com/rx-angular/rx-angular/tree/main/libs/template/docs/concepts.md)
+  - [Performance issues in Angular](https://github.com/rx-angular/rx-angular/tree/main/libs/template/docs/performance-issues.md)
+  - [Reactive context](https://github.com/rx-angular/rx-angular/tree/main/libs/template/docs/reactive-context.md)
 
-## Experimental features
+## Description
 
-Additionally, @rx-angular/template provides some experimental optimization tools which in general will give you more control
-about what changes are leading to re-renderings.
+**@rx-angular/template** is nothing less than a revolution in `ChangeDetection` for angular applications.
```
Comment (edbzn) [inline libs/template/README.md:91]:
```suggestion
**@rx-angular/template** is nothing less than a revolution in `ChangeDetection` for Angular applications.
```

File: README.md [hunk 8]
```diff
@@ -2,35 +2,49 @@
...
 > Expect no migration scripts for any change in `experimental` folders
+
+## Version Compatibility
+
+| Angular                | RxJS                 | @rx-angular/state | @rx-angular/template | @rx-angular/cdk     |
+| ---------------------- | -------------------- | ----------------- | -------------------- | ------------------- |
+| `^12.0.0` or `^13.0.0` | `^6.5.5` or `^7.4.0` | `> 1.4.6`         | `> 1.0.0-beta.29`    | `> 1.0.0-alpha.10`  |
+| `^11.0.0`              | `^6.5.5`             | `<= 1.4.6`        | `<= 1.0.0-beta.29`   | `<= 1.0.0-alpha.10` |
+
+Regarding the compatibility to RxJs, we generally stick to the compatibilities of the angular framework itself.
```
Comment (edbzn) [inline README.md:32]:
```suggestion
Regarding the compatibility with RxJS, we generally stick to the compatibilities of the Angular framework itself.
```

File: README.md [hunk 9]
```diff
@@ -2,35 +2,49 @@
...
+All the packages support RxJs versions `^6.5.5` || `^7.4.0`.
```
Comment (edbzn) [inline README.md:33]:
```suggestion
All the packages support RxJS versions `^6.5.5` || `^7.4.0`.
```

File: README.md [hunk 10]
```diff
@@ -2,35 +2,49 @@
...
+For more information about the compatibilities of angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3)
```
Comment (edbzn) [inline README.md:34]:
```suggestion
For more information about the compatibilities of Angular itself see this [gist](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3).
```

File: libs/state/README.md [hunk 11]
```diff
@@ -8,17 +8,86 @@
...
 
+## Key features
+
+- Slim and intuitive API
+- Automated subscription handling
+- Intuitive way for handling ViewModels
+- Connect any Observable source to the state
+- Partial state updates
+- Reactive state selection
+- Lazy state (no BehaviourSubject)
```
Comment (edbzn) [inline libs/state/README.md:61]:
```suggestion
- Lazy state (no BehaviorSubject)
```

**Review by edbzn — COMMENTED** [review]
I left a few minor comments.

**Comment by Karnaukhov-kh** [comment]
Thank you 👍. All added :)

**Review by edbzn — APPROVED** [review]

**Review by hoebbelsB — APPROVED** [review]
great stuff @Karnaukhov-kh

**Comment by hoebbelsB** [comment]
@edbzn should we squash this one?

**Pull request closed without merging**